src/tests/class_tests/openms/source/PeakIntegrator_test.cpp

```diff
@@ -145,74 +150,91 @@ END_SECTION
 
 START_SECTION(estimateBackground())
 {
-  MSChromatogram chromatogram;
-  setup_toy_chromatogram(chromatogram);
-  double left = 2.477966667;
-  double right = 3.01895;
-  double background = 0.0;
   Param params = ptr->getParameters();
 
   params.setValue("baseline_type", "base_to_base");
   params.setValue("integration_type", "intensity_sum");
   ptr->setParameters(params);
-  background = ptr->estimateBackground(chromatogram, left, right);
-  TEST_REAL_SIMILAR(background, 123446.661339019)
+  ptr->estimateBackground(chromatogram, left, right);
+  TEST_REAL_SIMILAR(ptr->getBackgroundArea(), 123446.661339019)
+  TEST_REAL_SIMILAR(ptr->getBackgroundHeight(), 16657.6971368377)
 
   params.setValue("baseline_type", "vertical_division");
   params.setValue("integration_type", "intensity_sum");
   ptr->setParameters(params);
-  background = ptr->estimateBackground(chromatogram, left, right);
-  TEST_REAL_SIMILAR(background, 50217)
+  ptr->estimateBackground(chromatogram, left, right);
+  TEST_REAL_SIMILAR(ptr->getBackgroundArea(), 50217)
+  TEST_REAL_SIMILAR(ptr->getBackgroundHeight(), 16657.6971368377)
 
   params.setValue("baseline_type", "base_to_base");
   params.setValue("integration_type", "trapezoid");
   ptr->setParameters(params);
-  background = ptr->estimateBackground(chromatogram, left, right);
-  TEST_REAL_SIMILAR(background, 1140.392865964)
+  ptr->estimateBackground(chromatogram, left, right);
+  TEST_REAL_SIMILAR(ptr->getBackgroundArea(), 1140.392865964)
+  TEST_REAL_SIMILAR(ptr->getBackgroundHeight(), 16657.6971368377)
 
   params.setValue("baseline_type", "vertical_division");
   params.setValue("integration_type", "trapezoid");
   ptr->setParameters(params);
-  background = ptr->estimateBackground(chromatogram, left, right);
-  TEST_REAL_SIMILAR(background, 476.606316373)
+  ptr->estimateBackground(chromatogram, left, right);
+  TEST_REAL_SIMILAR(ptr->getBackgroundArea(), 476.606316373)
+  TEST_REAL_SIMILAR(ptr->getBackgroundHeight(), 16657.6971368377)
 }
 END_SECTION
 
 START_SECTION(integratePeak())
 {
-  // inputs
-  MSChromatogram chromatogram;
-  setup_toy_chromatogram(chromatogram);
-  const double left = 2.477966667;
-  const double right = 3.01895;
-
   ptr->setIntegrationType("intensity_sum");
   ptr->integratePeak(chromatogram, left, right);
   cout << "intensity_sum: " << endl;
   TEST_REAL_SIMILAR(ptr->getPeakArea(), 6768778)
   TEST_REAL_SIMILAR(ptr->getPeakHeight(), 966489.0)
-  TEST_REAL_SIMILAR(ptr->getPeakApexPosition(), 2.7045)
+  TEST_REAL_SIMILAR(ptr->getPeakApexRT(), 2.7045)
```

Comment:
The name changes help make the code much more readable...at least to me.  We will see what the OpenMS team thinks.